### ISSUE TITLE
USER STORY: Model optimization V4

### ISSUE THREAD

**Issue by Huddy2022** [Huddy2022/milestone-project-mildew-detection-in-cherry-leaves#23]
As a **data scientist** I want to **train the CNN model using the pre-processed dataset with re-sizes images 100 x 100 and optimize the hyperparameters min value 64, max value 128, step 16, filters 64, 128 & 256, dropout 0.5, with values 1e-3, 1e-4, 1e-5, on a Relu activation, loss of binary_crossentropy and add regularizers L1 & L2 to achieve the best performance,** so that **the model can accurately classify cherry leaves as healthy or containing powdery mildew.**
